src/common/HeartbeatMap.cc

```diff
@@ -169,7 +170,7 @@ void HeartbeatMap::check_touch_file()
   if (path.length() && is_healthy()) {
     int fd = ::open(path.c_str(), O_WRONLY|O_CREAT|O_CLOEXEC, 0644);
     if (fd >= 0) {
-      ::utimes(path.c_str(), NULL);
+      ::utime(path.c_str(), NULL);
       ::close(fd);
```

Comment:
On FreeBSD: for utime() and utimes()
This interface is obsoleted by utimensat(2) because it is not accurate to
     fractions of a second.
So if you change, perhaps use on that is not made obsolete

Reply:
Nicest would be to find something that works on Posix and Windows, without those routines being made obsolete.
I guess they are not going aways, since they are abundantly used. But still

Reply:
> I guess they are not going aways, since they are abundantly used.

Probably it's best if we send a separate patch to handle that.